common/app/model/content.scala

```diff
@@ -267,6 +267,14 @@ class Content protected (val apiContent: ApiContentWithMeta) extends Trail with
     }
   }
 
+// This is a fix for the hanging letters on long section/series labels on the leftCol breakpoint.
+  lazy val areLabelsLong: Boolean = {
+    val sectionLabelList: Seq[String] = sectionLabelName.split(" ").toSeq
+    val blogLabelList: Seq[String] = blogOrSeriesTag.map(_.name.split(" ").toSeq).getOrElse(Nil)
+    val filteredList: Seq[String] = sectionLabelList.filter(_.length > 10) ++ blogLabelList.filter(_.length > 12)
```

Comment:
Feels more consistent for all longer labels to shrink down rather than ones that just fit a quite strict and odd criteria.

I don't think we should be doing this at all, but if we are we should keep the criteria really simple.